pipeline_steps/common.groovy

```diff
@@ -354,6 +354,16 @@ def prepareConfigs(Map args){
 
 def prepareRpcGit(Map args){
   dir("/opt/rpc-openstack"){
+
+    /* if job is triggered by PR, then we need to set RPC_REPO and
+       RPC_BRANCH using the env vars supplied by ghprb.
+    */
+    if ( env.ghprbPullId != null ){
+      env.RPC_REPO = "https://github.com/${env.ghprbGhRepository}.git"
+      env.RPC_BRANCH = "origin/pr/${env.ghprbPullId}/merge"
```

Comment:
gah `branch = env.RPC_BRANCH` will need to be above the inner if, otherwise it will override the branch from the pull request.